apps/projects/test/Projects.test.js

```diff
@@ -1156,6 +1170,122 @@ contract('Projects App', accounts => {
       })
     })
 
+    context('decoupled project', () => {
+      let repoId
+      beforeEach(async () => {
+        await app.setRepo('1', true, 'abc')
```

Comment:
We should use real repo ids as in other test cases to avoid false negatives when repoId length does not find in the contract field

Reply:
Projects will be using integers for decoupled repoIds going forward. They'll still get hashed by the frontend before loading them in, but that's really jus a messier magic number